localstack/services/dynamodb/resource_providers/aws_dynamodb_globaltable.py

```diff
@@ -0,0 +1,392 @@
+                }
+
+                create = {k: v for k, v in create.items() if v is not None}
+
+                replicas_to_create.append({"Create": create})
+
+                request.aws_client_factory.dynamodb.update_table(
+                    ReplicaUpdates=replicas_to_create, TableName=model["TableName"]
+                )
+
+            return ProgressEvent(
+                status=OperationStatus.IN_PROGRESS,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+
+        status = request.aws_client_factory.dynamodb.describe_table(TableName=model["TableName"])[
+            "Table"
+        ]["TableStatus"]
+        if status == "ACTIVE":
+            return ProgressEvent(
+                status=OperationStatus.SUCCESS,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+
+        return ProgressEvent(
+            status=OperationStatus.IN_PROGRESS,
+            resource_model=model,
+            custom_context=request.custom_context,
+        )
+
+    def read(
+        self,
+        request: ResourceRequest[DynamoDBGlobalTableProperties],
+    ) -> ProgressEvent[DynamoDBGlobalTableProperties]:
+        """
+        Fetch resource information
+
+        IAM permissions required:
+          - dynamodb:Describe*
+          - application-autoscaling:Describe*
+          - cloudwatch:PutMetricData
+        """
+        raise NotImplementedError
+
+    def delete(
+        self,
+        request: ResourceRequest[DynamoDBGlobalTableProperties],
+    ) -> ProgressEvent[DynamoDBGlobalTableProperties]:
+        """
+        Delete a resource
+
+        IAM permissions required:
+          - dynamodb:Describe*
+          - application-autoscaling:DeleteScalingPolicy
+          - application-autoscaling:DeleteScheduledAction
+          - application-autoscaling:DeregisterScalableTarget
+          - application-autoscaling:Describe*
+          - application-autoscaling:PutScalingPolicy
+          - application-autoscaling:PutScheduledAction
+          - application-autoscaling:RegisterScalableTarget
+        """
+
+        model = request.desired_state
+        if not request.custom_context.get(REPEATED_INVOCATION):
+            request.custom_context[REPEATED_INVOCATION] = True
+            request.aws_client_factory.dynamodb.delete_table(TableName=model["TableName"])
+
+            return ProgressEvent(
+                status=OperationStatus.IN_PROGRESS,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+
+        try:
+            request.aws_client_factory.dynamodb.describe_table(TableName=model["TableName"])
+            return ProgressEvent(
+                status=OperationStatus.IN_PROGRESS,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+        except Exception:
```

Comment:
Might be a little bit too generic. E.g. for actual (non-notfound) errors we do want to fail here instead of returning success